README.md

```diff
@@ -22,7 +20,20 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Configure slack_logger with a file inside of your config/initializers directory.
+It needs three configuration variables.
+Channel references the slack channel you would like your notices sent to.
+Token is used with the slack ruby client to connect to your slack environment.
+Enabled takes a block, which the gem uses to know what messages to send to your slack client.
+Example:
+
+```ruby
+SlackLogger.configure do |c|
+c.channel = '#slack-errors-channel'
```

Comment:
Sorry to be a pest about this but could you indent the lines inside the block by 2 spaces?